src/App.jsx

```
@@ -0,0 +1,45 @@
+  const projectId = import.meta.env.VITE_CLARITY_PROJECT_ID;
+  if (projectId) {
+      Clarity.init(projectId);
+      console.log(`Clarity initialized with project ID: ${projectId}`);
+  } else {
+      console.warn('Clarity project ID is not set. Please check your environment variables.');
+  }
+
+  return (
+    <>
+      <div>
+        <a href="https://vite.dev" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)